packages/react-data-grid/src/RowUtils.ts

```diff
@@ -1,8 +1,8 @@
 import { RowData } from './common/types';
 
-export function get(row: RowData, property: string) {
-  if (typeof row.get === 'function') {
-    return row.get(property);
+export function get<R extends {}>(row: R, property: keyof R) {
+  if (typeof (row as RowData).get === 'function') {
```

Comment:
I do not think we have a use case where `row.get` is a function. Let's test this and remove this api